components/Users.js

```diff
@@ -8,14 +8,15 @@ const Contributors = ({ users }) => {
       </div>
     );
   }
-  return users.map((user) => (
+  return users.map((user, index) => (
     <ProfileCard
+      key={index}
       username={user.name}
       avatarUrl={user.avatarUrl}
       socials={[
-        { type: "github", username: user.github },
-        { type: "twitter", username: user.twitter },
-        { type: "blog", username: user.blogUrl },
+        { type: 'github', username: user.github },
+        { type: 'twitter', username: user.twitter },
+        { type: 'blog', username: user.blogUrl },
       ]}
     />
   ));
```

Comment:
Instead of putting the key value to index, Can we have a unique id that is coming from the user data?
I heard from some developers, that it is not good to use index for key value.

Reply:
Yes, it is not a good idea to this. But this was only added in to make sure eslint checks pass. This PR tries not to change any code that does not affect performance or DX